qiskit_ibm_runtime/quantum_program/quantum_program.py

```diff
@@ -219,34 +219,34 @@ def append(
         *,
         samplex: Samplex | None = None,
         circuit_arguments: np.ndarray | None = None,
-        samplex_arguments: dict[str, np.ndarray] | None = None,
-        shape: tuple[int, ...] | None = None,
+        samplex_arguments: dict[str, Any] | None = None,
+        samplex_shape: tuple[int, ...] | None = None,
         chunk_size: int | None = None,
     ) -> None:
         """Append a new :class:`QuantumProgramItem` to this program.
 
         Args:
             circuit: The circuit of this item.
-            samplex: An (optional) samplex to draw random parameters for the circuit.
+            samplex: A samplex to draw random parameters for the circuit.
```

Comment:
We don't specify about optional arguments that they're optional, it's encoded in the `| None` of the type hints.